common/src/main/java/org/apache/falcon/metadata/EntityRelationshipGraphBuilder.java

```diff
@@ -80,7 +80,7 @@ public void addClusterEntity(Cluster clusterEntity) {
         Vertex clusterVertex = addVertex(clusterEntity.getName(), RelationshipType.CLUSTER_ENTITY);
 
         addUserRelation(clusterVertex);
-        addColoRelation(clusterEntity.getColo(), clusterVertex);
+        addColoRelation(clusterEntity.getColo(), clusterVertex, RelationshipLabel.CLUSTER_COLO);
```

Comment:
addColoRelation already sets correct relationship label. You wont have to pass it as arg.

        addEdge(fromVertex, coloVertex, RelationshipLabel.CLUSTER_COLO.getName());